### ISSUE TITLE
Use EDITOR env

### ISSUE THREAD

**Issue by tmtvl** [nige123/jmp.nigelhamilton.com#7]
A number of cli utilities use the EDITOR environment variable to choose an editor to launch (e.g. visudo, sudoedit, systemctl --edit,...). Maybe we can check if it's defined during install and set the initial config accordingly. That should also help prevent jmp config from launching nano if it isn't installed.

**Comment by nige123** [comment]
jmp relies on opening an editor at a specific line number. Unfortunately editors don't use consistent command line options to open at a specific line.

**Comment by tmtvl** [comment]
Yeah, but there are a few predefined, so couldn't we compare %*ENV{'EDITOR'} to the existing declarations and go from there?

**Comment by nige123** [comment]
Yes - I'm open to that. Pull requests welcome! ;-)
